packages/utils/types/organizations.ts

```diff
@@ -27,6 +29,35 @@ export type ConfigureVotingFormType = {
   minutes: string;
 };
 
+export const ZodRoleObject = z.object({
```

Comment:
please don't capitalize zod schema as they are objects
```suggestion
export const zodRoleObject = z.object({
```